purchase_sale_link/__openerp__.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# © 2017 Akretion
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+{'name': 'Purchase Sale Link',
+ 'version': '8.0.1.0.0',
+ 'author': 'Akretion,Odoo Community Association (OCA)',
+ 'license': 'AGPL-3',
+ 'category': 'Purchase Management',
+ 'depends': ['sale',
+             'purchase'
+             ],
+ 'website': 'http://www.akretion.com',
```

Comment:
https://github.com/OCA/purchase-workflow

Reply:
Hi! I updated the small changes you suggested, I vould be glad if you can review again. Thank you !